scripts/check-base-images.sh

```diff
@@ -7,9 +7,33 @@ set -euo pipefail
 
 source "${TOOLS_LIB}/libcli.source"
 
-# We exclude "base_version", which we use for base images that are built from this repo in the same version
+python_cli=(/usr/bin/env python3)
 
-if find cluster/images/ -name Dockerfile -exec grep -Hn "FROM" {} + | grep -v sha | grep -v base_version; then
-  _error "docker images must pin base images to a specific digest"
-fi
+# Use virtual environment for Python dependencies
```

Comment:
Please don't, the dependencies should come from nix.

Reply:
You should be able to add it in here https://github.com/hyperledger-labs/splice/blob/main/nix/shell.nix#L61

Reply:
done